src/uu/install/src/install.rs

```diff
@@ -748,6 +748,20 @@ fn perform_backup(to: &Path, b: &Behavior) -> UResult<Option<PathBuf>> {
 /// Returns an empty Result or an error in case of failure.
 ///
 fn copy_file(from: &Path, to: &Path) -> UResult<()> {
+    // fs::copy fails if destination is a invalid symlink.
+    // so lets just remove all existing files at destination before copy.
+    match fs::remove_file(to) {
+        Ok(()) => {}
+        Err(e) if e.kind() == std::io::ErrorKind::NotFound => { /* expected */ }
+        Err(e) => {
+            show_error!(
+                "Failed to remove existing file {}. Error: {:?}",
+                to.display(),
+                e
+            );
+        }
+    }
```

Comment:
can it be simplified this way ?

```suggestion
if let Err(e) = fs::remove_file(to) {
    if e.kind() != std::io::ErrorKind::NotFound {
        show_error!("Failed to remove existing file {}. Error: {:?}", to.display(), e);
    }
}


```

Reply:
true. done. Thanks :-)